src/gigamonkey/p2p/header_string.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2020 Katrina Knight
```

Comment:
It looks like we don't need this file at all. We can get rid of it.

Reply:
Removed